modules/programs/alacritty.nix

```diff
@@ -37,27 +40,35 @@ in {
         '';
         description = ''
           Configuration written to
-          {file}`$XDG_CONFIG_HOME/alacritty/alacritty.yml`. See
-          <https://github.com/alacritty/alacritty/blob/master/alacritty.yml>
-          for the default configuration.
+          {file}`$XDG_CONFIG_HOME/alacritty/${configFileName}`. See
```

Comment:
Unfortunately cannot do this since manual generation then depends on the user's configuration. Will need to be something like

``` markdown
{file}`$XDG_CONFIG_HOME/alacritty/alacritty.yml` or
{file}`$XDG_CONFIG_HOME/alacritty/alacritty.toml`,
where the latter is used for Alacritty 0.13 and later.
```

Reply:
Good point, fixed.